CONTRIBUTING.md

```diff
@@ -182,8 +185,8 @@ It's often better to check *before* contributing that you're setting off on the
 ## Fixing style guide inconsistencies
 
 Fantomas tries to keep up with the style guides, but as these are living documents, it can occur that something is listed in the style that Fantomas is not respecting.
-In this case, please create an issue using our [online tool](https://fsprojects.github.io/fantomas-tools/#/).  
```

Comment:
I think you've removed a couple of these in other PR's (perhaps automatically via tooling), but please don't remove these spaces.
They introduce a line break (`<br />` HTML element).

Reply:
Oh, sorry. That's a setting in my VS Code. I'll keep that in mind.